File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/config/WebSocketConfig.java
package org.javamaster.invocationlab.admin.config;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.socket.WebSocketHandler;
import org.springframework.web.socket.config.annotation.WebSocketConfigurer;
import org.springframework.web.socket.config.annotation.WebSocketHandlerRegistry;

/**
 * @author yudong
 * @date 2023/6/11
 */
@Configuration
public class WebSocketConfig implements WebSocketConfigurer {
    @Autowired
    private WebSocketHandler webSocketHandler;

    @Override
    public void registerWebSocketHandlers(WebSocketHandlerRegistry webSocketHandlerRegistry) {
        webSocketHandlerRegistry.addHandler(webSocketHandler, "/erdSyncProject").setAllowedOrigins("*");
    }
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/config/WebSocketConfig.kt
package org.javamaster.invocationlab.admin.config

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.annotation.Configuration
import org.springframework.web.socket.WebSocketHandler
import org.springframework.web.socket.config.annotation.WebSocketConfigurer
import org.springframework.web.socket.config.annotation.WebSocketHandlerRegistry

/**
 * @author yudong
 * @date 2023/6/11
 */
@Configuration
class WebSocketConfig : WebSocketConfigurer {
    @Autowired
    private lateinit var webSocketHandler: WebSocketHandler

    override fun registerWebSocketHandlers(webSocketHandlerRegistry: WebSocketHandlerRegistry) {
        webSocketHandlerRegistry.addHandler(webSocketHandler, "/erdSyncProject").setAllowedOrigins("*")
    }
}
